modules/planning/tasks/deciders/BUILD

```diff
@@ -79,12 +80,29 @@ cc_library(
     copts = ["-DMODULE_NAME=\\\"planning\\\""],
     deps = [
         ":decider_base",
+        ":path_decider_obstacle_utils",
         "//modules/planning/common:planning_context",
         "//modules/planning/common:planning_gflags",
         "//modules/planning/common:reference_line_info",
     ],
 )
 
+cc_library(
+    name = "path_decider_obstacle_utils",
+    srcs = [
+        "path_decider_obstacle_utils.cc",
+    ],
+    hdrs = [
+        "path_decider_obstacle_utils.h",
+    ],
+    copts = [
+        "-DMODULE_NAME=\\\"planning\\\"",
+    ],
```

Comment:
AFAIK, this definition is only necessary for the final cc_binary target.

Reply:
deleted.